main/src/ca/uwaterloo/flix/api/lsp/LanguageServer.scala

```diff
@@ -221,6 +223,7 @@ class LanguageServer(port: Int) extends WebSocketServer(new InetSocketAddress("l
       inputStream.close()
 
       packages += (uri -> items.toList)
+      println(s"packages: $packages")
```

Comment:
Remove debugging statement :)

Reply:
Oops - embarrassed I missed that. Sorry.